frontend/main.js

```diff
@@ -269,8 +273,26 @@ async function startApp () {
           'times-circle'
         )
       }
+
+      this.setBadgeOnTab()
     },
     computed: {
+      ...mapGetters(['ourUnreadMessages', 'unreadGroupNotificationCountFor']),
+      ...mapState(['contracts']),
+      ourUnreadMessagesCount () {
+        return Object.keys(this.ourUnreadMessages)
+          .map(cId => this.ourUnreadMessages[cId].mentionings.length)
+          .reduce((a, b) => a + b, 0)
+      },
+      ourNotificationsCount () {
+        return Object.keys(this.contracts)
+          .filter(cId => this.contracts[cId].type === 'gi.contracts/group')
+          .map(gId => this.unreadGroupNotificationCountFor(gId))
+          .reduce((a, b) => a + b, 0)
+      },
```

Comment:
This count doesn't seem to include private notifications. In case they must be counted too, then the `totalUnreadNotificationCount()` getter could be used.

Reply:
Yeah. `totalUnreadNotificationCount` is the right solution.